packages/vscode-extension/package.json

```diff
@@ -573,9 +573,9 @@
       {
         "title": "Teams Toolkit",
         "properties": {
-          "fx-extension.armSupportEnabled": {
+          "fx-extension.insiderPreview": {
             "type": "boolean",
-            "description": "Enable provision Azure resources with ARM templates. (Requires reload of VS Code)",
+            "description": "Enable insider preview features. (requires reload of VS Code)",
             "default": false
           },
```

Comment:
As we combined the feature flags, we need to remove the multi-env user setting.

Reply:
thanks, removed it in package.json